### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by djarecka**
@dbkeator I've started adding tests for `Project` and `Session`, but some tests will fail for asserts for `project.graph.get_records()` (e.g. [here](https://github.com/djarecka/PyNIDM/blob/more_tests/nidm/experiment/tests/test_experiment_basic.py#L89) )

Those tests don't fail if you run only one test at a time. That means that probably some class attributes are being changed. Do you know which part of the test can lead to it? Is it expected behavior?

### CODE REVIEW

**Comment by djarecka** [comment]
@dbkeator - this is [the global graph](https://github.com/incf-nidash/PyNIDM/blob/master/nidm/core/Constants.py#L84) that I was talking about during the call. So basically this graph is being updated (i.e. extended) every time the new project is created. I can create a simple class.

**Comment by djarecka** [comment]
I added [a simple class](https://github.com/incf-nidash/PyNIDM/pull/45/files#diff-ef834c311cdf89806cc21d85dfb4ff2bR84) to avoid  a global variable

File: nidm/core/Constants.py [hunk 1]
```diff
@@ -81,9 +81,9 @@
     q_graph.bind(name, namespace)
 
 # DBK Added - Empty graph using provtoolbox used to compute qnames
-p_graph = ProvDocument()
-for name, namespace in namespaces.items():
-    p_graph.add_namespace(name, namespace)
+class PGraph(ProvDocument):
```
Comment (satra) [inline nidm/core/Constants.py]: perhaps call this a `NIDMDocument`

Comment (djarecka) [reply nidm/core/Constants.py]: ok

**Comment by djarecka** [comment]
@satra, @dbkeator  - you can check quickly and merge

**Pull request merged**